### ISSUE TITLE
Support for WAVEFRONT_TOKEN and WAVEFRONT_ADDRESS still?

### ISSUE THREAD

**Issue by stephenchu** [snltd/wavefront-cli#19]
Are they still supported? I can't seem to get them to work using env vars, but only with a `~/.wavefront` ini file.

If so, can you point me to the code that does the parsing of such environment variables?

**Comment by snltd** [comment]
Hi Stephen. Thanks for the bug report. I thought the CLI used the SDK's credential mechanism, which does support the env vars, but for some reason I chose to duplicate part of the SDK code rather than using it properly. I'm on this, and should have a fix soon. Apologies!

**Comment by stephenchu** [comment]
Hey. No problem at all. I'm only a beneficiary from the great amount of work you put in wavefront-cli. Thank _you_.

**Comment by snltd** [comment]
This is fixed in master (pull #21), and in gem version 2.1.3. It now uses the credentials mechanism from the SDK. I think that was added to the SDK after the CLI was written. It also exposed a gap in the testing, which I've addressed. The env vars will override any config file values.

```
$ wf --version
2.1.3
$ wf proxy list
xxxxxxx-c294-4ebc-919a-56342cf4b945  Proxy on wavefront-blue
xxxxxxx-5171-45e0-8d31-5c980be889ea  Proxy on shark-wf-test
xxxxxxx-23c6-4fb9-8c8d-833625ab8f6f  Agent on shark-wavefront
$ WAVEFRONT_ENDPOINT=xxxxx.wavefront.com WAVEFRONT_TOKEN=xxxxxxx wf proxy list
xxxxxxx-8b8b-4a6b-a741-11bdf71f387b  Proxy on shark-wf-test
```

Thanks again for the bug report. Let me know if there are further issues.


Rob
